### PR TITLE
fix: made necessary changes to docusaurus

### LINKED ISSUE #115: fix: add required changes to docusaurus present in  /docs
### Is your feature request related to a problem? Please describe.
Certain changes are required in docusaurus present in the /docs directory
- Add the command `npm run serve` wherever required one can refer [this](https://github.com/anitab-org/open-source-programs-backend/blob/develop/osp-backend-docusaurus/README.md).
- The .features & .featureImage is not required, should be removed from [here](https://github.com/anitab-org/open-source-programs-web/blob/develop/docs/src/pages/styles.module.css).
- As the open source programs backend docusaurus has been published the link should be changed in [docusaurus.config.js](https://github.com/anitab-org/open-source-programs-web/blob/develop/docs/docusaurus.config.js)

### Describe the solution you'd like
- `npm run serve` will help checking with the local build during change
-  The 2 features mentioned above are not required
-  Correct link should be mentioned

### Additional context
npm
feature_not_required
doc_backend

### PR DESCRIPTION
**Pull request by ashwinijha6**
### Description
Made necessary changes to docusaurus :
- Add the command `npm run serve` wherever required in README.md in docs/
- Removed .features & .featureImage from [here](https://github.com/anitab-org/anitab-forms-web/blob/develop/docs/src/pages/styles.module.css)
- Changed the link for Backend docs in the [docusaurus.config.js](https://github.com/anitab-org/anitab-forms-web/blob/develop/docs/docusaurus.config.js)
- Removed exclamation from Home.md

Fixes #115 

### Type of Change:

- Code
- Quality Assurance
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] I have written Kotlin Docs whenever is applicable

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by ashwinijha6** [comment]
@keshakaneria  & @codesankalp could u'll pls review this PR

**Review by keshakaneria — DISMISSED** [review]
Looks good 🎉

File: docs/README.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ yarn build
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
 
+**Note**: You can use `npm run serve` to test your build locally.
```
Comment (codesankalp) [inline docs/README.md]:
```suggestion
**Note**: You can use `yarn serve` to test your build locally.
```

**Review by codesankalp — CHANGES_REQUESTED** [review]
I think adding a line for yarn also, will be a good option.

File: docs/README.md (hunk 1 above)
Comment (ashwinijha6) [reply docs/README.md]: @codesankalp could you pls explain why I should replace `npm run serve` with `yarn serve` ? I don't see much difference in it, but I guess `yarn` is much faster than `npm`.

Comment (codesankalp) [reply docs/README.md]: @ashwinijha6 You have added the same note line twice. As the first few lines of the README include `yarn start`, `yarn build` etc.. then according to the flow there should be `yarn serve` for testing the build locally.
I think it is a valid reason.

Comment (ashwinijha6) [reply docs/README.md]: Yes that was on purpose @codesankalp. I can do one thing that is change the command after `yarn build` to `yarn serve` & let the second command after `npm run build` be as it is OR remove the second command. What do you think?

**Comment by ashwinijha6** [comment]
@keshakaneria & @codesankalp just I have made the changes required

**Review by codesankalp — APPROVED** [review]

**Review by Amulya-coder — CHANGES_REQUESTED** [review]
@ashwinijha6 QA checks are failing please run the linter and formatter as mentioned in the readme.

**Comment by ashwinijha6** [comment]
Hey @Amulya-coder I did this error while running `npm run lint:fix` Should I perform this before running it `npm install --save-dev eslint-config-prettier`
eslint

**Comment by codesankalp** [comment]
@ashwinijha6 Delete your node_modules and package-lock.json.
and try npm install.

**Comment by ashwinijha6** [comment]
> @ashwinijha6 Delete your node_modules and package-lock.json.
> and try npm install.

Ok I'll try that @codesankalp

**Comment by isabelcosta** [comment]
Will close this due to lack of update. cc @codesankalp 
@ashwinijha6 feel free to claim this again if you're interested!

**Pull request closed without merging**